lib/src/nodes/config_node_path.cc

```diff
@@ -0,0 +1,17 @@
+#include <internal/nodes/config_node_path.hpp>
+
+using namespace std;
+
+namespace hocon {
+
+    config_node_path::config_node_path(path node_path, token_list tokens) :
+        _path(move(node_path)), _tokens(move(tokens)) { }
+
+    token_list config_node_path::get_tokens() const {
```

Comment:
This seems like it could return a `token_list const&`, to avoid copying the vector if we just need access to the contents.